.github/workflows/ci.yaml

```diff
@@ -444,3 +445,65 @@ jobs:
         asset_name: tanzu-apps-plugin${{ matrix.os-arch }}-${{ env.RELEASE_VERSION }}.tar.gz
         asset_content_type: application/gzip
 
+  publish-oci-artifact:
+    env:
+      TKG_CUSTOM_IMAGE_REPOSITORY: projects-stg.registry.vmware.com/tkg
```

Comment:
question: as this is a vmware server, should it come from a secret/config-var?

Reply:
do we need this for v0.28.0 of TF? have we tried removing it and see if the CI succeeds?

Reply:
okay, removed and succeeded